packages/flutter/lib/src/foundation/math.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Same as [num.clamp] but optimized for non-null [double].  This is roughly 4x
```

Comment:
nit, put the second sentence on a new line

Reply:
done